src/main/java/no/nav/pto/veilarbportefolje/domene/Bruker.java

```diff
@@ -180,7 +188,9 @@ public static Bruker of(OppfolgingsBruker bruker, boolean ufordelt, boolean erVe
                 .setHarUtelandsAddresse(harUtelandsAddresse)
                 .setHarUkjentBosted(bruker.isHarUkjentBosted())
                 .setBostedSistOppdatert(bruker.getBostedSistOppdatert())
-                .setHarAvvik14aVedtak(bruker.getAvvik14aVedtak() == null && bruker.getAvvik14aVedtak() != Avvik14aVedtak.INGEN_AVVIK);
+                .setAvvik14aVedtak(bruker.getAvvik14aVedtak())
+                .setInnsatsgruppe(PortefoljeMapper.mapTilInnsatsgruppe(valueOfOrNull(Kvalifiseringsgruppe.class, bruker.getKvalifiseringsgruppekode())))
```

Comment:
small suggestion to improve readability: move this to small private method:
`PortefoljeMapper.mapTilInnsatsgruppe(valueOfOrNull(Kvalifiseringsgruppe.class, bruker.getKvalifiseringsgruppekode()))`

Reply:
Fjerner disse feltene heilt da vi ikkje hadde behov for dei i frontenden likevel.